src/lib/jsonParseV2/utilsParse.h

```diff
@@ -38,6 +38,22 @@ struct Opt {
   bool ok() { return error.empty(); }
 };
 
+
+
+/*
+ * getBoolMust -  get a mandatory bool from the rapidjson node
+ */
+Opt<bool> getBoolMust(const rapidjson::Value& parent, const char* field,const std::string& description = "");
```

Comment:
Leave them be.
We will probably use them some day.
If you want, use #if 0 / #endif to not occupy and code in the executable